weed/s3api/s3api_bucket_handlers_test.go

```diff
@@ -4,37 +4,34 @@ import (
 	"github.com/seaweedfs/seaweedfs/weed/s3api/s3err"
 	"testing"
 	"time"
-
-	"github.com/aws/aws-sdk-go/aws"
-	"github.com/aws/aws-sdk-go/service/s3"
 )
 
 func TestListBucketsHandler(t *testing.T) {
 
 	expected := `<?xml version="1.0" encoding="UTF-8"?>
-<ListAllMyBucketsResult xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><Owner><DisplayName></DisplayName><ID></ID></Owner><Buckets><Bucket><CreationDate>2011-04-09T12:34:49Z</CreationDate><Name>test1</Name></Bucket><Bucket><CreationDate>2011-02-09T12:34:49Z</CreationDate><Name>test2</Name></Bucket></Buckets></ListAllMyBucketsResult>`
+<ListAllMyBucketsResult><Owner xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><ID xmlns="http://s3.amazonaws.com/doc/2006-03-01/"></ID></Owner><Buckets xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><Bucket xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><Name xmlns="http://s3.amazonaws.com/doc/2006-03-01/">test1</Name><CreationDate xmlns="http://s3.amazonaws.com/doc/2006-03-01/">2011-04-09T12:34:49Z</CreationDate></Bucket><Bucket xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><Name xmlns="http://s3.amazonaws.com/doc/2006-03-01/">test2</Name><CreationDate xmlns="http://s3.amazonaws.com/doc/2006-03-01/">2011-02-09T12:34:49Z</CreationDate></Bucket></Buckets></ListAllMyBucketsResult>`
```

Comment:
Is there any option to generate code from xsd without the `xmlns`? Seems wasteful to transmit repeated unnecessary bytes. 

(`ListAllMyBuckets` is ok since it is not called often, but other operations should try to avoid the repetitions. )